bin/simbol

```diff
@@ -1,40 +1,49 @@
 #!/bin/bash
 # vim: tw=0:ts=4:sw=4:et:ft=bash
 
-e=0
+let -i e=0
+
+#. Source user-overrides if any
 [ ! -f ${HOME}/.simbolrc ] || source ${HOME}/.simbolrc
+
+#. If the bash-shell override has been defined, exec into that first
 if [ ${#SIMBOL_SHELL} -gt 0 ]; then
-    if [ -x "${SIMBOL_SHELL}" ]; then
+    if [ -x "${SIMBOL_SHELL}" -a -f "${SIMBOL_SHELL}" ]; then
         if [ "${SIMBOL_SHELL}" != "${BASH}" ]; then
-            exec ${SIMBOL_SHELL} $0 $@
+            exec ${SIMBOL_SHELL} -$- "$0" "$@"
```

Comment:
So `$-` is the set of options with which bash was instantiated.  This ensures that if I run `bash -x simbol` for example, and I have an override for `SIMBOL_SHELL`, then it will execute the new shell with the `-x`.